shopgate/pwa
# Fixed scroll position maintenance

# Description

When navigating from a scrolled category page to the product page and back, you wouldn't be brought back to the scrolled position you left the page. This has been fixed.

## Type of change

Please add an "x" into the option that is relevant:

- [ x ] Bug Fix :bug: (non-breaking change which fixes an issue)
- [  ] New Feature :rocket: (non-breaking change which adds functionality)
- [  ] Breaking Change :boom: (fix or feature that would cause existing functionality to not work as expected)
- [  ] Polish :nail_care: (Just some cleanups)
- [  ] Docs :memo: (Changes in the documentations)
- [  ] Internal :house: Only relates to internal processes.

## How to test it

Navigate to a category page with many products, scroll for a bit and click on a product. Then navigate back. You should be at the same position, you have left before.
